docs/guides/vqa_guide/estimators.rst

```diff
@@ -4,46 +4,162 @@
+    - :func:`group_comeasureable_terms_greedy <orquestra.vqa.grouping.group_comeasureable_terms_greedy>`
+- :func:`perform_context_selection <orquestra.vqa.estimation.context_selection.perform_context_selection>`
+
+
+If you look closely, you might notice that some of these methods do not actually follow the preprocessor interface. Let's take a look at `allocate_shots_proportionally`. It takes two things as input - list of estimation tasks and total number of shots. So why do we say it's kindof a preprocessor? Cause it's easy to make it into a preprocessor using `partial`:
+
+
+.. literalinclude:: ../../examples/estimators_guide.py
+    :start-after: >> Tutorial code snippet: script showing how to use partials
+    :lines: 2-4
+    :language: python
+
+
+Utility functions
+-----------------
+
+We have also created a couple of methods which make working with estimators easier, here's a brief overview:
+
+Let's start with factories of estimation tasks. What is a factory? Factory is a method which creates some objects according to a particular recipe, given the inputs. It's useful when you don't know ahead of time exactly what the parameters of your object should be. In case of variational algorithms we can't create circuits ahead, as we don't know what their parameters will be - they are updated on the fly. If we don't know what the circuis are, we also can't create our estimation tasks. But we can create an object (factory), which will generate those estimation tasks. This all happens inside the cost function and fortunately you don't have worry too much about how exactly. Two most important factories are :func:`substitution_based_estimation_tasks_factory <orquestra.vqa.cost_function.cost_function.substitution_based_estimation_tasks_factory>` and :func:`dynamic_circuit_estimation_tasks_factory <orquestra.vqa.cost_function.cost_function.dynamic_circuit_estimation_tasks_factory>`. The main difference between them is that the first one creates a circuit once and then just subtitutes symbols with parameters, while the other one creates circuits on the fly.
+
+Some other tools:
+
+- :func:`evaluate_estimation_circuits <orquestra.quantum.estimation.evaluate_estimation_circuits>` - substitutes symbols with numerical parameters for all the circuits in the estimation tasks 
+- :func:`split_estimation_tasks_to_measure <orquestra.quantum.estimation.split_estimation_tasks_to_measure>` - this function splits a given list of EstimationTask into two: one that contains EstimationTasks that should be measured, and one that contains EstimationTasks with constants or with 0 shots.
+- :func:`evaluate_non_measured_estimation_tasks <orquestra.quantum.estimation.evaluate_non_measured_estimation_tasks>` - this returns the expectation value for the tasks which don't need to be measured
+
+Example
+=======
+
+Let's see how this plays out in an actual example. For the sake of example, let's say we want to run VQE using the following techniques:
```

Comment:
Remove "For the sake of example" and say "For example"

Reply:
Fixed in https://github.com/zapatacomputing/orquestra-core/pull/60/commits/0b3d2fd9b5438795d3280b6ecad3885360af1ba2